cloudbuild/cloudbuild-pr.yaml

```diff
@@ -34,7 +34,7 @@ steps:
   id: Lint and test forseti-security charts
   args: ["ct", "lint-and-install", "--config=cloudbuild/ct.yaml"]
   env: ["CLOUDSDK_CONTAINER_USE_APPLICATION_DEFAULT_CREDENTIALS=true"]
-
+timeout: 900s
```

Comment:
What is this timeout based on?

Reply:
The build was timing out at ten minutes.  I was just testing.  This will be removed before approval.